app/models/megaphone/episode.rb

```diff
@@ -16,15 +16,15 @@ class Episode < Integrations::Base::Episode
 
     CREATE_ATTRIBUTES = CREATE_REQUIRED + %i[pubdate pubdate_timezone author link explicit draft
       subtitle summary background_image_file_url background_audio_file_url pre_count post_count
-      insertion_points guid pre_offset post_offset expected_adhash original_filename original_url
+      insertion_points guid expected_adhash original_filename original_url
```

Comment:
we never use these old pre_offset post_offset attributes, as we set cerpoints, but, sending them as nil was causing 500 errors, so just leave them out.